### DISCUSSION TITLE
Can't find the signalk-node-red in the Signal K AppStore

### DISCUSSION THREAD

**Discussion by markante** [SignalK/signalk-node-red#35] [Q&A] [answered]
Hi,
I Can't find the signalk-node-red in the Signal K AppStore. How can I install it ?

Thanks for your help
Mark

**Comment by sbender9** [comment, marked as answer]
It's in the App Store.

**Reply by markante** [reply]
Now I found it. The Dropdown was only on "NEW". Thank you very much
